.github/workflows/all.yml

```diff
@@ -0,0 +1,178 @@
+name: Deps, Lint, Test, Build, Publish
+on:
+  # Enable manually triggering this workflow via the API or web UI
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - feat/ci-overhaul  # TODO: Remove
+    tags:
+      - v*
+  pull_request:
+
+env:
+  APP_NAME: "k6"
+  DOCKER_IMAGE_ID: ${{ github.repository }}
+  # Force pure Go DNS resolver, see https://golang.org/pkg/net/#hdr-Name_Resolution
+  # Needed for some of our tests (TestErrorCodes).
+  GODEBUG: netdns=go+1
```

Comment:
This can probably be removed if we decide to leave `TestErrorCodes` disabled, although we should figure out a way to make it pass...

Reply:
Can you link me where it fails for all platforms ?

Reply:
I couldn't find a failure on all platforms, so I might be misremembering that, but [here's the Linux one](https://github.com/imiric/k6/runs/928285449).

Reply:
Resolved by c9b1b9ba.